js-modern-syntax/js-modern-syntax_destructuring-arrays/index.js

```diff
@@ -6,41 +6,54 @@ console.clear();
 
 const item = ["Egg", 0.25, 12];
 
+export const [name, price, quantity] = item;
+
 // example: export const [value1, value2] = array;
 
 // EXERCISE 2
 // Use array destructuring to extract the variables "firstNameOfChris", "lastNameOfChris" and "ageOfChris".
 
 const personChris = [12, "Chris", "Owen"];
 
+export const [, firstNameOfChris, lastNameOfChris, ageOfChris = 12] =
```

Comment:
age is already declared as 12 in the personChris array above, so no need to do  ageOfChris = 12. So, below will work
`export const [, firstNameOfChris, lastNameOfChris, ageOfChris ] =  personChris;`

Reply:
> age is already declared as 12 in the personChris array above, so no need to do ageOfChris = 12. So, below will work `export const [, firstNameOfChris, lastNameOfChris, ageOfChris ] = personChris;`

still error , for mine does not work with this way. i will ask Swen or Laya tomorrow.